src/ballet/sha256/fd_sha256.c

```diff
@@ -349,16 +349,15 @@ fd_sha256_fini( fd_sha256_t * sha,
 
   /* Unpack the result into md (annoying bswaps here) */
 
-  uint * hash = (uint *)_hash;
-  hash[0] = fd_uint_bswap( state[0] );
-  hash[1] = fd_uint_bswap( state[1] );
-  hash[2] = fd_uint_bswap( state[2] );
-  hash[3] = fd_uint_bswap( state[3] );
-  hash[4] = fd_uint_bswap( state[4] );
-  hash[5] = fd_uint_bswap( state[5] );
-  hash[6] = fd_uint_bswap( state[6] );
-  hash[7] = fd_uint_bswap( state[7] );
-  return _hash;
+  state[0] = fd_uint_bswap( state[0] );
+  state[1] = fd_uint_bswap( state[1] );
+  state[2] = fd_uint_bswap( state[2] );
+  state[3] = fd_uint_bswap( state[3] );
+  state[4] = fd_uint_bswap( state[4] );
+  state[5] = fd_uint_bswap( state[5] );
+  state[6] = fd_uint_bswap( state[6] );
+  state[7] = fd_uint_bswap( state[7] );
+  return fd_memcpy( _hash, state, 32 );
```

Comment:
```suggestion
  return memcpy( _hash, state, 32 );
```

Normal memcpy is preferred here because it allows the compiler to generate faster unaligned SIMD stores. 
fd_memcpy is actually slower here because the `state` variable is in SIMD registers, not in memory.